### ISSUE TITLE
UU: Tom nedtrekksmeny for språkversjoner på enkelte sider

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-enonicxp-frontend#568]
Rapport i Siteimprove:
https://my2.siteimprove.com/Inspector/1002489/A11Y/Page?pageId=25525448885&impmd=6DD5538FAB0E174F64D81B4146785183#/sia-r68/failed/ME9PVzdqc0g=;MDg2YjYxOWE=
Eksempelside:
https://www.nav.no/no/nav-og-samfunn/kontakt-nav/utbetalinger/snarveier/ferie-og-feriepenger

**Comment by PerOlavM** [comment]
Sjekk om nedtrekksmenyen tømmes ved avpublisering av "søstersider".

**Comment by PerOlavM** [comment]
HTML for nedtrekkmeny (nav+ul) opprettes bare hvis oppkoblede språkversjoner er publisert.

**Comment by PerOlavM** [comment]
Løst i dekoratøren: 
https://github.com/navikt/nav-dekoratoren/pull/1039
